Makefile

```diff
@@ -410,8 +410,20 @@ ui:
 		rm -rf ./pkg/extensions/build;\
 		cp -R $$tdir/zui/build ./pkg/extensions/;\
 	else\
-		curl -fsSL https://github.com/project-zot/zui/releases/download/$(ZUI_VERSION)/zui.tgz -o zui.tgz;\
-		tar xvzf zui.tgz -C ./pkg/extensions/;\
-		rm zui.tgz;\
+		curl --fail --head https://github.com/project-zot/zui/releases/download/$(ZUI_VERSION)/zui.tgz;\
+		if [ $$? -ne 0 ]; then\
+  			git clone https://github.com/project-zot/zui.git;\
```

Comment:
Let's clone in a temporary folder. See line 403

Reply:
Also git clone **that** version - $(ZUI_VERSION)?

Reply:
I have made the changes